statsmodels/stats/tests/test_anova_rm.py

```diff
@@ -131,3 +134,62 @@ def test_repeated_measures_collinearity():
 def test_repeated_measures_unbalanced_data():
     assert_raises(ValueError, AnovaRM, data.iloc[1:48, :], 'DV', 'id',
                   within=['A', 'B'])
+
+
+def test_repeated_measures_aggregation():
+    df1 = AnovaRM(data, 'DV', 'id', within=['A', 'B', 'D']).fit()
+    df2 = AnovaRM(data.append(data), 'DV', 'id', within=['A', 'B', 'D'],
+                  aggregate_func=np.mean).fit()
+
+    assert_frame_equal(df1.anova_table, df2.anova_table)
+
+
+def test_repeated_measures_aggregation_one_subject_duplicated():
+    df1 = AnovaRM(data, 'DV', 'id', within=['A', 'B', 'D']).fit()
+    df2 = AnovaRM(data.append(data.loc[data['id'] == '1', :]).reset_index(),
+                  'DV', 'id', within=['A', 'B', 'D'],
+                  aggregate_func=np.mean).fit()
+
+    assert_frame_equal(df1.anova_table, df2.anova_table)
+
+
+def test_repeated_measures_aggregate_func():
+    assert_raises(ValueError, AnovaRM, data.append(data), 'DV', 'id',
+                  within=['A', 'B', 'D'])
+
+    m1 = AnovaRM(data.append(data), 'DV', 'id', within=['A', 'B', 'D'],
+                 aggregate_func=np.mean)
+    m2 = AnovaRM(data.append(data), 'DV', 'id', within=['A', 'B', 'D'],
+                 aggregate_func=np.median)
+
+    assert m1.aggregate_func is not m2.aggregate_func
```

Comment:
Use numpy testing assert_, no plain assert in unit tests
Also afaics the results in m1 and m2 should be the same, additional test?